packages/@holaplexui-react/src/components/Form.tsx

```diff
@@ -309,3 +303,50 @@ const FormCheckbox = forwardRef(function FormCheckbox(
 });
 
 Form.Checkbox = FormCheckbox;
+
+interface DragDropImageProps {
+  onChange: (file: File) => void;
+  value: File | null;
+  children: JSX.Element;
+  className?: string;
+}
+
+const DragDropImage = ({ onChange, value, className, children }: DragDropImageProps) => {
+  const [file, setFile] = useState<File | null>(value);
+  const { getInputProps, isDragActive } = useDropzone({
+    accept: {
+      'image/png': ['.png'],
+      'image/jpeg': ['.jpeg', '.jpg'],
+      'image/svg': ['.svg'],
+    },
+    maxSize: 2 * 1024 * 1024,
+    onDrop: (acceptedFiles) => {
+      if (acceptedFiles.length > 0) {
+        const file = acceptedFiles[0];
+        setFile(file);
+        onChange(file);
+      }
+    },
+  });
+  return (
+    <div className={clsx('relative', className)}>
+      <input {...getInputProps()} />
+      <div
+        className={clsx('dragdropimage', {
+          'dragdropimage-drag-active': isDragActive,
+          'dragdropimage-empty': !file,
+        })}
+      >
+        {file ? (
+          <div className="dragdropimage-image-container">
+            <img src={URL.createObjectURL(file)} alt={file.name} className="dragdropimage-image" />
```

Comment:
The value should just be an imageurl string not a file can make the the alt the image url as well.